packages/react-icons/package.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "@coveord/plasma-react-icons",
+    "version": "1.0.0",
+    "description": "Plasma iconography exposed as react components",
+    "main": "./dist/index.js",
+    "types": "./dist/index.d.ts",
+    "scripts": {
+        "build": "node ./bin/index.js && tsc --project tsconfig.json"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/coveo/plasma.git",
+        "directory": "packages/react-icons"
+    },
+    "keywords": [
+        "plasma",
+        "icons",
+        "react"
+    ],
+    "author": "",
+    "license": "ISC",
+    "bugs": {
+        "url": "https://github.com/coveo/plasma/issues"
+    },
+    "homepage": "https://github.com/coveo/plasma/packages/react-icons#readme",
+    "files": [
+        "dist"
+    ],
+    "devDependencies": {
+        "@coveord/plasma-tokens": "workspace:*",
```

Comment:
The version for this package will be sync with plasma-token? :eyes: 
If Plasma-Token bumps, so does this one?

Reply:
Exactly, would it be better to commit the generated files so that we can review the changes?

Reply:
:thinking: In theory, the react icons components should changes only if the icons from the libraries changes right? 
So we would not have a commit with 780 icons each times?  :laughing:  If so then I think it's fine to commit them :thinking:

Reply:
Yes, but we would commit and review generated code that we cannot edit manually anyway. If we don't commit them we will still see what changed in the svg files of the plasma-tokens package.

Reply:
Certes :+1: